scripts/server.sh

```diff
@@ -4,14 +4,6 @@ set -e
 
 echo "Run the RWD Docker container."
 
-if [ -z "$RWD_DATA" ]; then
```

Comment:
For consistency in error messages, consider removing the same for `./scripts/console.sh`